cmake/configure/configure_20_trilinos.cmake

```diff
@@ -446,20 +446,28 @@ macro(feature_trilinos_configure_external)
   if(${DEAL_II_TRILINOS_WITH_TPETRA})
     if(_tpetra_inst_double)
       set(DEAL_II_EXPAND_TPETRA_TYPES "double")
-      set(DEAL_II_EXPAND_TPETRA_VECTOR_DOUBLE "LinearAlgebra::TpetraWrappers::Vector<double>")
+      set(DEAL_II_EXPAND_TPETRA_VECTOR_DOUBLE
+        "LinearAlgebra::TpetraWrappers::Vector<double, MemorySpace::Host>"
+        "LinearAlgebra::TpetraWrappers::Vector<double, MemorySpace::Default>")
```

Comment:
What happens here if you do not have a device enabled in Kokkos, do you still get two different instantiations increasing the library size significantly?

Reply:
Yes, you get two different instantiations. I don't know if this increases the library size significantly. Ultimately, the same Tpetra instantiations are used in this case.

Reply:
I am seeing
```
248M	libdeal_II.9.7.0-pre.dylib
317M	libdeal_II.g.9.7.0-pre.dylib
```
with this pull request and
```
240M	libdeal_II.9.7.0-pre.dylib	
307M	libdeal_II.g.9.7.0-pre.dylib
```
without. So like 3%.